docs/standard/native-interop/abi-support.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Native interoperability ABI support - .NET
+description: Defines the current support for interoperability with various ABIs.
+ms.date: 10/28/2024
+---
+# Native interoperability ABI support
+
+The [Application Binary Interface](https://wikipedia.org/wiki/Application_binary_interface) (ABI) is the interface which run-times and operating systems express low-level binary details. This can include calling conventions (that is, how parameters are passed and results returned), exception handling, as well as symbol mangling. The below list contains the names of languages, run-times and even general technologies since these names tend to represent what users will be working with when looking for guidance on interoperability.
```

Comment:
```suggestion
The [Application Binary Interface](https://wikipedia.org/wiki/Application_binary_interface) (ABI) is the interface which run-times and operating systems express low-level binary details. This can include calling conventions (that is, how parameters are passed and results returned), exception handling, as well as symbol mangling. The below list contains the names of languages, run-times and even general technologies since these names tend to represent scenarios when looking for guidance on interoperability.
```
"what users will be working with" does not sound like a style that we use in documentation. (The suggestion is my best guess - I will defer to the doc style experts to phrase it properly.)

Reply:
Agree. I'm hoping they can help too :)